strimzi-kafka.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: strimzi-kafka
+  version: 0.40.0
+  epoch: 0
+  description: Kafka addons for Strimzi
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - openjdk-17-jre
+
+environment:
+  environment:
+    JAVA_HOME: /usr/lib/jvm/java-17-openjdk
+    JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8
+    KAFKA_HOME: /opt/kafka
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - gradle
+      - openjdk-17
+      - maven
+      - make
+      - zip
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/strimzi/strimzi-kafka-operator
+      tag: ${{package.version}}
+      expected-commit: 1b5ce127db873913923f267ed4e213c719179a59
+
+  - runs: |
+      cd ./docker-images/artifacts
+      mvn dependency:copy-dependencies ${MVN_ARGS} -f ./kafka-thirdparty-libs/3.7.x/pom.xml
+      mkdir -p ./binaries/kafka-thirdparty-libs
+      rm -f ./binaries/kafka-thirdparty-libs/${version_lib}.zip
+      zip -j ./binaries/kafka-thirdparty-libs/${version_lib}.zip kafka-thirdparty-libs/3.7.x/target/dependency/*
+      cd -
+
+      mkdir -p ${{targets.destdir}}/${KAFKA_HOME}/libs/
+
+      set -e
+      for component in kafka-agent mirror-maker-agent tracing-agent; do
+        cd ./${component}
+        make java_install
+        cp /home/build/.m2/repository/io/strimzi/${component}/${{package.version}}/${component}-${{package.version}}.jar ${{targets.destdir}}/${KAFKA_HOME}/libs/
```

Comment:
These all get placed under /root so this copy fails. I believe HOME needs to be set to /home/build in the environment to resolve this